add GitHub configuration for issues, pull requests and general support (#1019)
rauc/rauc: .github/pull_request_template.md

<!--
Thank you for your pull request!

If this is your first pull request for RAUC, please read:
https://rauc.readthedocs.io/en/latest/contributing.html

> Please describe what it changes, i.e. fix this bug and how, add a feature, fix documentation…

Reviewer: ```suggestion
Please describe what it changes, e.g. fixes a bug (and how), adds a feature, fixes documentation…
```
